StudyBuddy/Managers/FileManager.cs

```diff
@@ -1,32 +1,31 @@
 ﻿using System.Globalization;
+using CsvHelper;
+using CsvHelper.Configuration;
 using StudyBuddy.Abstractions;
 using StudyBuddy.Models;
 using StudyBuddy.ValueObjects;
-using CsvHelper.Configuration;
-using CsvHelper;
 
 namespace StudyBuddy.Managers
 {
     public class FileManager
     {
         private readonly IUserManager _userManager;
-        static bool loaded = false;
+        private readonly bool _loaded;
         private const string CsvDelimiter = ";";
+
         public FileManager(IUserManager userManager)
         {
             _userManager = userManager;
 
-            if (!loaded)
+            if (_loaded)
```

Comment:
Loaded is false by default?

Reply:
For less nesting do something like

```cs
if (_loaded)
{
    return;
}

LoadUsersFromCsv("test.csv");
_loaded = true;
```

Reply:
_loaded must remain static btw, otherwise it will upload the same profiles 2,3,4...times as long as you keep switching between profile creation and profile explore